spec/pundit_spec.rb

```diff
@@ -413,6 +421,10 @@
       expect(controller.authorize(post)).to be(post)
     end
 
+    it "returns the record when passed record with namespace " do
+      expect(Pundit.authorize(user, [:project, comment], :update?)).to be(comment)
+    end
```

Comment:
This looks identical to the `.authorize` spec. Should this be `controller.authorize` instead?